README.md

```diff
@@ -46,14 +47,23 @@ module SomeGem
   end
 
   class SomeClass
-    def something_dangerous
-      # ...
-    rescue => ex
-      SomeGem.escalate(ex, "I was doing something dangerous and an exception was raised")
+    def something_dangerous(key)
+      # code here...
+    rescue => exception
+      SomeGem.escalate(exception,
+                       "Exception raised in SomeGem::SomeClass#something_dangerous",
+                       context: { key: key })
```

Comment:
I decided to keep `context`, but make it an explicit keyword arg throughout `escalate`. I think it will be handy to pair with the `location_message` just above.